hugo/content/about.md

```diff
@@ -0,0 +1,73 @@
+---
+title: About
+toc: true
+---
+
+# About the project
+
+Mumble is a free, open source, low latency, high quality voice chat application.
+
+## Source Code & Development
+
+You find the Source Code and further instructions regarding Building and Development in our [GitHub Repo](https://github.com/mumble-voip).
+
+## Licenses
+
+<!-- https://github.com/mumble-voip/mumble/blob/master/LICENSE -->
+
+<!-- more licenses? should be as correct and complete as possible -->
+
+# Who we are
+
+<!-- Title could also be: Developers, Team etc. -->
+
+<!-- improve -->
+
+We are a team of voluntary developers - both professional and non-professional coders - from different countries - including Germany, Italy, USA and more.
+
+<!--
+Link to Github Team page:
+https://github.com/orgs/mumble-voip/people
+
+-->
+
+<!-- Distinction between active and non-active developers? -->
```

Comment:
I'd rather not do that since this is very prone to become outdated. You could link to https://github.com/mumble-voip/mumble/graphs/contributors though. There the user can filter for the time interval they are interested in and then evaluate for themselves who was active in that period.

Reply:
Well that is not really ideal.
See above :point_up:.